### PR TITLE
Fixed #24950 -- Added unicode_literals to models.py in app template

### PR DESCRIPTION
**Pull request by MarkusH**
Thanks to Tim Graham for the patch

### CODE REVIEW

File: django/core/management/templates.py [hunk 1]
```diff
@@ -104,6 +104,7 @@ def handle(self, app_or_project, name, target=None, **options):
             base_directory: top_dir,
             'docs_version': get_docs_version(),
             'django_version': django.__version__,
+            'unicode_literals': '' if six.PY3 else 'from __future__ import unicode_literals\n\n'
```
Comment (timgraham) [inline django/core/management/templates.py]: missing trailing comma 
also not sure if a test is needed?

**Comment by MarkusH** [comment]
Added a small test without introducing too much more overhead to the (already) slow admin_script tests

**Comment by timgraham** [comment]
Looks good. I guess the proper semantics would be to use `six.PY2` since we probably wouldn't `unicode_literals` on Python 4 if such a thing existed -- not that it will be a practical problem.

**Comment by MarkusH** [comment]
Merged in 5d428908218e3692d1bc7e4ada6235029ae9e45c.

**Pull request closed without merging**